CHANGELOG.rst

```diff
@@ -2,6 +2,17 @@
 Changelog
 =========
 
+TBR
+---
+
+* Support for the new features in ``web_poet>=0.2.0`` for supporting additional 
```

Comment:
Reminder to confirm this before merging since the anticipated `web_poet` version release might change.